packages/docmanager-extension/src/index.ts

```diff
@@ -633,11 +604,6 @@ function addLabCommands(
     return docManager.findWidget(pathMatch[1]);
   };
 
-  // Closes an array of widgets.
-  const closeWidgets = (widgets: Array<Widget>): void => {
-    widgets.forEach(widget => widget.close());
-  };
-
   // Find the tab area for a widget within a specific dock area.
   const findTab = (
```

Comment:
I think this function is no longer used (and the compiler doesn't catch it because it uses itself, maybe?)

Reply:
This may be outdated, but `closeWidgets` is used in a few spots still...

Reply:
I mean `findTab`. Is that used anywhere?

Reply:
It's used in `tabAreaFor`

Reply:
Didn't you move that to `application-extension`?

Reply:
Yep, I see it in docmanager-extension too. Removed!